### PR TITLE
♻️ Visualise body in stacktraces from S3 HTTP errors

### PR DESCRIPTION
**Pull request by GitHK**
## What do these changes do?

When debugging responses from AWS, it is very hard to figure out what is wrong with them. Seeing only `400 Bad Request` is not resolutive since this status code is raised for various reasons [see here](https://docs.aws.amazon.com/AmazonS3/latest/API/ErrorResponses.html).

You are also interested in the body of the response, which contains more details about the error [see here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingRESTError.html).

These changes allow you to log the body of the response in the stacktrace  as well in case of HTTPError (status >= 400).

## Related issue/s

- https://github.com/ITISFoundation/osparc-simcore/issues/3402
- https://github.com/ITISFoundation/osparc-issues/issues/638
## Checklist

- [x] Unit tests for the changes exist

### CODE REVIEW

**Review by sanderegg — APPROVED** [review]
cool, thanks for changing this

File: packages/simcore-sdk/src/simcore_sdk/node_ports_common/file_io_utils.py [hunk 1]
```diff
@@ -47,6 +49,50 @@ class UploadableFileObject:
...
+    ):
+        super().__init__(
+            request_info,
+            history,
+            code=code,
+            status=status,
+            message=message,
+            headers=headers,
+        )
+        self.body = body
```
Comment (pcrespov) [inline packages/simcore-sdk/src/simcore_sdk/node_ports_common/file_io_utils.py:72]: MINOR: can you please add the links in the description here? I think it justifies better why would you want to expose the body in an error response

File: packages/simcore-sdk/src/simcore_sdk/node_ports_common/file_io_utils.py [hunk 2]
```diff
@@ -47,6 +49,50 @@ class UploadableFileObject:
...
+
+    def __str__(self) -> str:
+        return (
+            f"status={self.status}, "
+            f"message={self.message}, "
+            f"url={self.request_info.real_url}, "
+            f"body={self.body=}"
```
Comment (pcrespov) [inline packages/simcore-sdk/src/simcore_sdk/node_ports_common/file_io_utils.py:79]: TYPO: ``f"body={self.body}"``  (without last ``=``)

**Review by pcrespov — APPROVED** [review]
Please check minor comments. thx!

**Pull request merged**